engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -208,6 +219,42 @@ private void createPlayer(String shipName) {
         objectManager.resetDelays();
     }
 
+    /**
+     * Creates and spawns the players mercenaries from their JSON file.
+     */
+    private void createMercs() {
+        String fileName = "mercenaries.json";
```

Comment:
This string "mercenaries.json" should be a static variable, since it's also used in the loading logic.